packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_panel.dart

```diff
@@ -140,3 +149,69 @@ class _PropertyEditorConnectedPanelState
     );
   }
 }
+
+class _PropertyEditorFooter extends StatelessWidget {
+  const _PropertyEditorFooter();
+
+  static const _footerHeight = 25.0;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final colorScheme = theme.colorScheme;
+    final documentationLink = _documentationLink();
+    return Container(
+      decoration: BoxDecoration(
+        color: colorScheme.surface,
+        border: Border(top: BorderSide(color: Theme.of(context).focusColor)),
+      ),
+      height: _footerHeight,
+      padding: const EdgeInsets.symmetric(vertical: densePadding),
+      child: Row(
+        crossAxisAlignment: CrossAxisAlignment.end,
+        children: [
+          if (documentationLink != null)
+            Padding(
+              padding: const EdgeInsets.only(left: denseSpacing),
+              child: _DocsLink(
+                documentationLink: documentationLink,
+                color: colorScheme.onSurface,
+              ),
+            ),
+          const Spacer(),
+          ReportFeedbackButton(color: colorScheme.onSurface),
+        ],
+      ),
+    );
+  }
+
+  String? _documentationLink() {
+    final queryParams = DevToolsQueryParams.load();
+    final isEmbedded = queryParams.embedMode.embedded;
+    if (!isEmbedded) return null;
+    const uriPrefix = 'https://docs.flutter.dev/tools/';
+    const uriHash = '#property-editor';
+    return '$uriPrefix${queryParams.ide == 'VSCode' ? 'vs-code' : 'android-studio'}$uriHash';
```

Comment:
we should verify with the docs maintainers that there is a way to ensure we don't have to duplicate the content even if it is documented at two different locations.

Reply:
sg to clean up later.

Reply:
Yes, if nothing else I was planning on having these sections link to a shared Property Editor section